Aarch64 Android compiles as `armv8-a` and don't have so much extensions.

--- a/third_party/dav1d/config/linux/arm64/config.h
+++ b/third_party/dav1d/config/linux/arm64/config.h
@@ -29,7 +29,7 @@
 
 #define ARCH_X86_64 0
 
-#define AS_ARCH_LEVEL armv8.6-a+crc
+// #define AS_ARCH_LEVEL armv8.6-a+crc
 
 #define CONFIG_16BPC 1
 
@@ -45,15 +45,15 @@
 
 #define HAVE_ASM 1
 
-#define HAVE_AS_ARCHEXT_DOTPROD_DIRECTIVE 1
+// #define HAVE_AS_ARCHEXT_DOTPROD_DIRECTIVE 1
 
-#define HAVE_AS_ARCHEXT_I8MM_DIRECTIVE 1
+// #define HAVE_AS_ARCHEXT_I8MM_DIRECTIVE 1
 
-#define HAVE_AS_ARCHEXT_SVE2_DIRECTIVE 1
+// #define HAVE_AS_ARCHEXT_SVE2_DIRECTIVE 1
 
-#define HAVE_AS_ARCHEXT_SVE_DIRECTIVE 1
+// #define HAVE_AS_ARCHEXT_SVE_DIRECTIVE 1
 
-#define HAVE_AS_ARCH_DIRECTIVE 1
+// #define HAVE_AS_ARCH_DIRECTIVE 1
 
 #define HAVE_AS_FUNC 0
 
@@ -63,13 +63,13 @@
 
 #define HAVE_DLSYM 1
 
-#define HAVE_DOTPROD 1
+// #define HAVE_DOTPROD 1
 
 #define HAVE_ELF_AUX_INFO 0
 
 #define HAVE_GETAUXVAL 1
 
-#define HAVE_I8MM 1
+// #define HAVE_I8MM 1
 
 #define HAVE_IO_H 0
 
@@ -87,9 +87,9 @@
 
 #define HAVE_PTHREAD_SET_NAME_NP 0
 
-#define HAVE_SVE 1
+// #define HAVE_SVE 1
 
-#define HAVE_SVE2 1
+// #define HAVE_SVE2 1
 
 #define HAVE_SYS_TYPES_H 1
 
